content/influxdb/cloud-dedicated/reference/internals/storage-engine.md

```diff
@@ -38,11 +40,29 @@ queries, and is optimized to reduce storage cost.
 
 ## Storage engine components
 
+- [Router](#router)
 - [Ingester](#ingester)
 - [Querier](#querier)
 - [Catalog](#catalog)
 - [Object store](#object-store)
 - [Compactor](#compactor)
+- [Garbage collector](#garbage-collector)
+
+### Router
+
+The Router (also known as the Ingest Router) parses and shards incoming line
+protocol and then routes each shard to [Ingesters](#ingester).
+To ensure write durability, the Router replicates each shard to two of the
+available Ingesters.
+
+##### Router scaling strategies
+
+The Router can be scaled both [vertically](#vertical-scaling) and
+[horizontally](#horizontal-scaling).
+Horizontal scaling increases write throughput and is typically the most
+effective scaling strategy for the Router.
+Vertical scaling (specifically increased CPU) improves the Router's ability to
+parse incoming line protocol under heavy load.
```

Comment:
Both vertical and horizontal scaling improve performance under heavy load, so I'd drop "under heavy load" here.

Vertical scale can help parse *individual requests with lower latency*.

Vertical scaling is easier on the catalog (Postgres or service) because the Router's catalog query cache reduces the need for catalog client requests.